upf/upf_pfcp.c

```diff
@@ -1980,6 +1989,7 @@ pfcp_update_apply (upf_session_t * sx)
 	    upf_pfcp_session_stop_urr_time (&urr->time_threshold, now);
 	    upf_pfcp_session_stop_urr_time (&urr->time_quota, now);
 	    upf_pfcp_session_stop_urr_time (&urr->traffic_timer, now);
+	    upf_pfcp_session_stop_urr_time (&urr->quota_validity_time, now);
```

Comment:
I would put this after time_quota.

Side Note: it is somewhat strange that the quota_holding_time and monitoring_time is not cleared as well. But that would be a topic for another PR.